src/main/java/com/stripe/model/Topup.java

```diff
@@ -0,0 +1,213 @@
+    Long created;
+    String currency;
+    String description;
+    Long expectedAvailabilityDate;
+    String failureCode;
+    String failureMessage;
+    Map<String, String> metadata;
+    Boolean livemode;
+    Source source;
+    String statementDescriptor;
+    String status;
+
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getObject() {
+        return object;
+    }
+
+    public void setObject(String object) {
+        this.object = object;
+    }
+
+    public Integer getAmount() {
+        return amount;
+    }
+
+    public void setAmount(Integer amount) {
+        this.amount = amount;
+    }
+
+    public String getBalanceTransaction() {
+        if (this.balanceTransaction == null) {
+            return null;
+        }
+        return this.balanceTransaction.getId();
+    }
+
+    public void setBalanceTransaction(String balanceTransactionID) {
+        this.balanceTransaction = setExpandableFieldID(balanceTransactionID, this.balanceTransaction);
+    }
+
+    public BalanceTransaction getBalanceTransactionObject() {
+        if (this.balanceTransaction == null) {
+            return null;
+        }
+        return this.balanceTransaction.getExpanded();
+    }
+
+    public void setBalanceTransactionObject(BalanceTransaction c) {
+        this.balanceTransaction = new ExpandableField<BalanceTransaction>(c.getId(), c);
+    }
+
+    public Long getCreated() {
+        return created;
+    }
+
+    public void setCreated(Long created) {
+        this.created = created;
+    }
+
+    public String getCurrency() {
+        return currency;
+    }
+
+    public void setCurrency(String currency) {
+        this.currency = currency;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Long getExpectedAvailabilityDate() {
+        return expectedAvailabilityDate;
+    }
+
+    public void setExpectedAvailabilityDate(Long expectedAvailabilityDate) {
+        this.expectedAvailabilityDate = expectedAvailabilityDate;
+    }
+
+    public String getFailureCode() {
+        return failureCode;
+    }
+
+    public void setFailureCode(String failureCode) {
+        this.failureCode = failureCode;
+    }
+
+    public String getFailureMessage() {
+        return failureMessage;
+    }
+
+    public void setFailureMessage(String failureMessage) {
+        this.failureMessage = failureMessage;
+    }
+
+    @Override
+    public Map<String, String> getMetadata() {
+        return metadata;
+    }
+
+    public void setMetadata(Map<String, String> metadata) {
+        this.metadata = metadata;
+    }
+
+    public Boolean getLivemode() {
+        return livemode;
+    }
+
+    public void setLivemode(Boolean livemode) {
+        this.livemode = livemode;
+    }
```

Comment:
Same thing here, the accessors for `livemode` should be before those for `metadata`.